.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,9 +3,9 @@
 ##### Contributor checklist
 
 - [ ] Included tests for the changes.
-- [ ] PR title is short, clear, and ready to be included in the user-facing changelog.
+- [ ] A newsfile is created in `changelog.d/` (see `changelog.d/README.md` for instructions) or the PR text says "no changelog needed".
 
 ##### Maintainer checklist
 
-- [ ] Verified one of these labels is present: `backwards incompatible`, `feature`, `enhancement`, `deprecation`, `bug`, `dependency`, `docs` or `skip-changelog` as they determine changelog listing.
+- [ ] If no changelog is needed, apply the `skip-changelog` label.
```

Comment:
We might need to record some guidelines for this (maybe, in a follow-up).

Reply:
I think this is okay for now? But if you have some idea on where/how to link, I'm happy to include a small change.

Reply:
In aiohttp, we have a bit of explanation right in the PR template: https://github.com/aio-libs/aiohttp/pull/11320. However, it's also possible to add a Markdown link to RTD (or GH README preview), just like in the following checkbox one line below.

Reply:
Sure, no rush to make it perfect in this PR.